builds.yml

```diff
@@ -123,16 +99,12 @@ features:
       # Assets that are exclusively included in this feature and ignored in others
       # Supports globs
       - ./{app,shared,ui}/**/beta/**
-  build-mmi:
-    assets:
-      - src: ./app/build-types/mmi/images/
-        dest: images
-      - ./{app,shared,ui}/**/institutional/**
   build-flask:
     assets:
       - src: ./app/build-types/flask/images/
         dest: images
       - ./{app,shared,ui}/**/flask/**
+  build-mmi:
```

Comment:
The build fence processing we do requires this to exist; I'm guessing you discovered this, and re-added to to get the build to pass.

Thoughts on leaving the build conifg and build scripts as-is until the production code is removed? i.e. revert the changes to this file and `development/build/config.js`, and restore the build scripts to the README and `package.json`. For me it seems a bit more intuitive to remove the config alongside the code that uses it, and this does fall outside of what this PR is described as doing.

Reply:
Great point - running that up the pipeline now